myconverter/converter.py

```diff
@@ -0,0 +1,78 @@
+from PIL import Image
+import pyheif
+import fitz
+import shutil
+import os
+
```

Comment:
Code quality: Rearranging "import" portion into three separated parts. 
- First: Native library.
- Second: Non-native library through package manager (ex: numpy, etc), locally installed library.
- Third: Function which imported from the same code base.